src/brevitas/graph/equalize.py

```diff
@@ -192,7 +265,7 @@ def _combine_weights_bias(
         return weight.data
 
     bias = bias.data
-    weight = weight.reshape(weight.shape[0], -1)
+    weight = weight.data.reshape(weight.shape[0], -1)
```

Comment:
why do we need data?

Reply:
Coherence with respect to what we do with the bias I suppose